jd-geek-source/src/app/components/icons/empty-star-icon/empty-star-icon.component.ts

```diff
@@ -1,12 +1,14 @@
-import { Component, Output } from "@angular/core";
+import { Component, Input, Output } from "@angular/core";
+import { SizeProp } from "@fortawesome/fontawesome-svg-core";
 import { faStar } from "@fortawesome/free-regular-svg-icons";
 
 @Component({
   selector: "app-empty-star-icon",
-  template: "<fa-icon [icon]='faStar'></fa-icon>",
+  template: "<fa-icon [size]='iconSize' [icon]='faStar'></fa-icon>",
   styleUrls: ["./empty-star-icon.component.scss"],
 })
 export class EmptyStarIconComponent {
+  @Input() iconSize: SizeProp;
   faStar = faStar;
   @Output() outStar = faStar;
```

Comment:
is this being used?

Reply:
not at the moment.  This was a refactor, but I will be using this when I revisit some of the cosmetic issues that I know exist.